k8s/celery/celery-config.yml

```
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: celery-config
+data: 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)